state/machine.go

```diff
@@ -347,6 +347,30 @@ func (m *Machine) IsManual() (bool, error) {
 	return false, nil
 }
 
+// IsIsolated returns true if the machine networking should be ran by fan
+// TODO this returns true for all containers on models with FAN configured,
+// it should use value inherited from '--isolated' cmdline flag
+func (m *Machine) IsIsolated() bool {
```

Comment:
Surely a machine is isolated if it only has machine local for fan local addresses right?

Reply:
This is a dummy function for until we have the proper --isolated flag.